layouts/shortcodes/featured/slide.html

```diff
@@ -21,40 +21,38 @@
 {{ $id := partial "helper/new-id" . }}
 
 
-<figure
-  class="group w-5/6 flex-none cursor-pointer snap-center px-2 pb-2 focus:outline-none sm:px-8"
-  :class="{'cursor-pointer': currentSlide !== +$el.dataset.slideId }"
-  x-intersect:enter.full="currentSlide = +$el.dataset.slideId"
-  data-slide-id="{{ .Ordinal }}"
-  @focus="center($event.currentTarget)"
-  tabindex="0"
-  data-ga-category="slide:{{ .Ordinal }}"
-  x-report-click
+<li
+  x-bind="disableNextAndPreviousButtons"
+  role="option"
+  class="h-full w-11/12 flex-[0_0_auto] snap-center px-0 font-bold md:w-11/12 [&:first-child]:snap-start [&:last-child]:snap-end"
 >
-  <div class="relative h-0 w-full pb-16x9">
-    <picture
-      class="
+  <figure
+    class="group flex-none cursor-pointer snap-center pb-2 focus:outline-none"
```

Comment:
Can you make it so it only uses cursor-pointer over the left/right select buttons?